### PR TITLE
Add Azure Back

### PR DESCRIPTION
**Pull request by sweeneyskirt-sl**
### Description
Adding the Azure CI doc back with a deprecated tag and manual redirect.

### Motivation and Context
Because our docs inadvertently got indexed prematurely, removing this file has caused a 404 because it shows up in Google search results.

### Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation fix (typos, incorrect content, missing content, etc.)

### Checklist

- [ ] I have read the [contributing](https://github.com/saucelabs/sauce-docs/blob/master/CONTRIBUTING.MD) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by spider-sauce — APPROVED** [review]
LGTM 👍 , but why is the pipeline failing?

File: docs/ci/azure.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+id: azure
+title: Sauce Labs with Azure
+sidebar_label: Azure
+---
+
+<p><span className="sauceRed"">PAGE DEPRECATED</span> |</small></p>
```
Comment (sweeneyskirt-sl) [inline docs/ci/azure.md:7]:
```suggestion
<p><span className="sauceRed">PAGE DEPRECATED</span> |</small></p>
```

File: docs/ci/azure.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
+id: azure
+title: Sauce Labs with Azure
+sidebar_label: Azure
+---
+
+<p><span className="sauceRed">PAGE DEPRECATED</span> |</small></p>
```
Comment (sweeneyskirt-sl) [inline docs/ci/azure.md:7]:
```suggestion
<p><span className="sauceRed">PAGE DEPRECATED</span></small></p>
```

File: docs/ci/azure.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+---
+id: azure
+title: Sauce Labs with Azure
+sidebar_label: Azure
+---
+
+<p><span className="sauceRed">PAGE DEPRECATED</span></small></p>
```
Comment (sweeneyskirt-sl) [inline docs/ci/azure.md:7]:
```suggestion
<p><span className="sauceRed">PAGE DEPRECATED</span></p>
```

**Pull request merged**